perllib/FixMyStreet/Cobrand/Bristol.pm

```diff
@@ -13,6 +13,10 @@ Bristol is a unitary authority, with its own Open311 server.
 package FixMyStreet::Cobrand::Bristol;
 use parent 'FixMyStreet::Cobrand::Whitelabel';
 
+use Moo;
+with 'FixMyStreet::Roles::Open311Alloy';
```

Comment:
Note this is already included on the `bristol-main-alloy-integration-add-csv-role` branch